app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> int | float:

```

Comment:
If you read the `README.md` file, you can see that we will never get `float` as the type of return value. We always get `int`

Reply:
since there are no division in those formulas, I agree